Add regular (e.g. not API) doc for cross namespace (#27975)
hashicorp/vault: website/content/docs/enterprise/namespaces/configure-cross-namespace-access.mdx

---
layout: docs
page_title: Configure cross namespace access without hierarchical relationships
description: >-
  Set up cross namespace access without hierarchical relationships for Vault Enterprise.
---

# Configure cross namespace access

Using the `group_policy_application` endpoint, you can enable secrets sharing
across multiple independent namespaces.

> Historically, any policies attached to a group would only apply when the

Reviewer: ```suggestion
Historically, any policies attached to an [identity group](/vault/docs/concepts/identity#identity-groups) would only apply when the
```

Vault token authorizing a request was created in the same namespace as that
group, or a descendent namespace.

> This endpoint allows relaxing that restriction. When the mode is set to the
> default, `within_namespace_hierarchy`, the historical behaviour is maintained.
> When set to `any`, group policies apply to all members of a group, regardless of
> what namespace the request token came from.

Reviewer: ```suggestion
This endpoint reduces the operational overhead by relaxing this restriction.
When the mode is set to the default, `within_namespace_hierarchy`, the
historical behaviour is maintained. When set to `any`, group policies apply to
all members of a group, regardless of what namespace the request token came
from.
```
